test/__snapshots__/index.test.js.snap

```diff
@@ -41,6 +41,8 @@ Build output to dist:
 exports[`fixtures ts-demo 1`] = `
 "ts-demo
   dist
+    car.d.ts
+    index.d.ts
```

Comment:
Why does it add **two** declaration files?

Reply:
I don't think typescript can generate a single declaration file when there is multiple source files

Reply:
Oh, right, LGTM! 👍